src/library_wasmfs_opfs.js

```diff
@@ -402,10 +402,14 @@ mergeInto(LibraryManager.library, {
   },
 
   _wasmfs_opfs_flush_access__deps: ['$wasmfsOPFSAccessHandles'],
-  _wasmfs_opfs_flush_access: async function(ctx, accessID) {
+  _wasmfs_opfs_flush_access: async function(ctx, accessID, errPtr) {
     let accessHandle = wasmfsOPFSAccessHandles.get(accessID);
-    // TODO: Error handling
-    await accessHandle.flush();
+    try {
+      await accessHandle.flush();
+    } catch {
+      let err = -{{{ cDefine('EIO') }}};
+      {{{ makeSetValue('errPtr', 0, 'err', 'i32') }}};
```

Comment:
I guess this adds `errPtr` since there wasn't an existing error to reuse? Just want to make sure I understand the convention.

Reply:
Yeah, exactly. That's what was happening before with `setSize` as well.